### ISSUE TITLE
Technique Validity Period and Contract hasDuration Period

### ISSUE THREAD

**Issue by paulakeen** [OP-TED/ePO#161]
We need to reflect on the meaning, definition and modelling of these two durations. See the diagram we have as per today and notes taken during the WG Meeting below the diagram:

![image](https://user-images.githubusercontent.com/16134316/53025245-d8c7fa80-3460-11e9-9d61-fe1cf35685ed.png)
Beware that the indivIuals of Framework Agreements that are Techniques are different from the individuals of Framework Agreements that are contracts. They are used for different purposes and in different moments. 

The lifecycle of Techniques are different from the legal life-cycle of Contracts. These two periods are used in different moments and different purposes, but they're dependent and rules need to be drafted to control that they are not inconsistent.

Examples: 

In other words,

the property "has Validity Period" is "reserved to" Technicques, to explain the valid duration of the technique, whilst the "has Duration" property domain is "Contract".

**Comment by muricna** [comment]
The predicates with the Period class need to be clearly verified and defined the version shown in the comment above does not correspond to the versions 2.0.1 and frozen branch 2.0.2.

**Comment by andreea-pasare** [comment]
Below is a diagram that presents how Period is modeled in ePO 3.0.1:

![image](https://user-images.githubusercontent.com/88081600/195003819-34f9dc46-5de9-4bb4-9302-c46838380881.png)

**Comment by andreea-pasare** [comment]
Rather than using different predicate, the concept Duration as been created.

In ePO v4.0.0-rc.1, the Period is modeled as below:
![image](https://github.com/OP-TED/ePO/assets/88081600/bcd6958c-9159-44c6-bf02-ece3fee427c1)

and the Duration is modeled as:
![image](https://github.com/OP-TED/ePO/assets/88081600/f3d91e9c-2bde-4127-9966-6702be448f0f)
